app/views/kaminari/_first_page.html.erb

```diff
@@ -0,0 +1,3 @@
+<li class="page-item">
```

Comment:
Why these?? Do we need to create them separately??

Reply:
![image](https://github.com/agarwalrashika00/quizathon/assets/56035062/c23993a9-7ec0-4f63-a40c-8626e45d3c61)
It looks like this if i donot add view files.

![image](https://github.com/agarwalrashika00/quizathon/assets/56035062/386b0fba-f6d1-4301-89ba-fa6e085d114e)
and it gets bootstrap classes if i add the view files